### PR TITLE
index copy and basic structure

### PR DESCRIPTION
**Pull request by tjmasterson**
@SStrausman @stephmarx to look at

### CODE REVIEW

File: app/views/welcome/index.html.erb [hunk 1]
```diff
@@ -1 +1,42 @@
...
+<div>
+  <p>
+    Our charity works to collect clothing, furniture, food, toys, books, hygiene products, and other miscellaneous materials for the homeless &amp; people in need.
+  </p>
+  <p>
+    WE ACCEPT ALL KINDS OF DONATIONS TO HELP THOSE IN NEED!
+  </p>
+  <ul>
+    <li>
+      Donations of food, hygiene products ,&nbsp;clothing, &amp;&nbsp;furniture &nbsp; are especially appreciated by the less fortunate
```
Comment (SStrausman) [inline app/views/welcome/index.html.erb]: Please change &s to and. In some cases they should be removed completely. For instance line 11 should be "products, clothing and furniture" 

Please change all caps to downcase.

line 11 needs a period.

line 14 is an unfinished thought.

21 recycling and redistributing 

please go through and check for proper capitalization like the words New and Used on line 21.

Please remove iframe and video.

remove everything after 501c3 line.

**Comment by tjmasterson** [comment]
@stephmarx @SStrausman all changes requested from review made

File: app/views/welcome/index.html.erb [hunk 2]
```diff
@@ -1 +1,28 @@
...
+  </p>
+  <p>
+    We accept all kinds of donations to help those in need!
+  </p>
+  <ul>
+    <li>
+      Donations of food, hygiene products,&nbsp;clothing,&nbsp;and furniture &nbsp; are especially appreciated by the less fortunate.
+    </li>
+    <li>
+      Resources that may help low income&nbsp;families,&nbsp;and those after leaving shelters.
```
Comment (SStrausman) [inline app/views/welcome/index.html.erb]: Please change &nbsp; characters to spaces.

**Comment by SStrausman** [comment]
Code reviewed.

**Pull request merged**